packages/flame/lib/src/game/game.dart

```diff
@@ -355,11 +358,17 @@ abstract mixin class Game {
     gameStateListeners.remove(callback);
   }
 
+  @internal
+  bool isInternalRefresh = false;
```

Comment:
This can be private instead of `@internal` now I think?

Reply:
Changed.